environment.yml

```diff
@@ -18,7 +18,7 @@ dependencies:
   - h5py
   - hdmf
   - hdf5
-  - pynwb
+  - pynwb>=1.5.1
```

Comment:
I was running into some issues previously when I had earlier version and found that upgrading my pynwb solved the issue, so that's why I added this requirement. I don't recall exactly what the error was though.

Reply:
Thanks!